odm2api/ODM2/models.py

```diff
@@ -750,6 +465,12 @@ class Results(Base):
     UnitsObj = relationship(Units)
     VariableObj = relationship(Variables)
 
+    __mapper_args__ = {
+        'polymorphic_on':ResultTypeCV,
+        'polymorphic_identity':'results',
+        'with_polymorphic':'*'
+    }
+
     def __repr__(self):
         return "<Results('%s', '%s', '%s', '%s', '%s')>" % (
             self.ResultID, self.ResultUUID, self.ResultTypeCV, self.ProcessingLevelID, self.ValueCount)
```

Comment:
Is it an explicit decision not to report all Results properties here? I realize there are a lot of properties, and that you didn't make any changes to this code block per se; I'm just wondering.